From 6bff03ab1a43ab69405631773f8c21f1d762793a Mon Sep 17 00:00:00 2001
From: Lulu Cheng <chenglulu@loongson.cn>
Date: Mon, 3 Jun 2024 06:05:20 +0000
Subject: [PATCH 086/133] LoongArch: Modify the output message string of the
 warning.

Fix bug for "error: spurious trailing punctuation sequence '.' in format [-Werror=format-diag]".

gcc/ChangeLog:

    * config/loongarch/loongarch-opts.cc: Modify the output message string
    of the warning.

Signed-off-by: Peng Fan <fanpeng@loongson.cn>
---
 gcc/config/loongarch/loongarch-opts.cc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/gcc/config/loongarch/loongarch-opts.cc b/gcc/config/loongarch/loongarch-opts.cc
index a76d0afbd38..6e08e16a4e6 100644
--- a/src/gcc/config/loongarch/loongarch-opts.cc
+++ b/src/gcc/config/loongarch/loongarch-opts.cc
@@ -378,8 +378,8 @@ fallback:
   t.cmodel = constrained.cmodel ? opt_cmodel : CMODEL_NORMAL;
   if (t.cmodel != CMODEL_NORMAL)
     {
-      warning (0, "%qs is not supported, now cmodel is set to 'normal'.",
-	       loongarch_cmodel_strings[t.cmodel]);
+      warning (0, "%qs is not supported, now cmodel is set to %qs",
+          loongarch_cmodel_strings[t.cmodel], "normal");
       t.cmodel = CMODEL_NORMAL;
     }
 
-- 
2.43.4

